docs/content/Ingestion-overview.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: doc_page
+---
+# Ingestion Overview
+
+There are a couple of different ways to get data into Druid. We hope to unify things in the near future, but for the time being
+the method you choose to ingest your data into Druid should be driven by your use case.
+
+## Streaming Data
+
+If you have a continuous stream of data, there are a few options to get your data into Druid. It should be noted that the current state of real-time ingestion in Druid,
+like many other popular stream infrastructure open source projects, does not guarantee exactly once processing. The real-time pipeline is meant to surface insights on
+ events as they are occurring. For an accurate copy of ingested data, an accompanying batch pipeline is required. We are working towards a streaming only word, but for
+ the time being, we recommended running a lambda architecture.
+
+### Ingest from a Stream Processor
+
+If you process your data using a stream processor such as Apache Samza or Apache Storm, you can use the [Tranquility](https://github.com/metamx/tranquility) library to manage
+your real-time ingestion. This setup requires using the indexing service for ingestion, which is what is used in production by many organizations that use Druid.
+
+### Ingest from Apache Kafka
+
+If you wish to ingest directly from Kafka using Tranquility, you will have to write a consumer that reads from Kafka and passes the data to Tranquility.
+The other options to use [standalone Realtime nodes]().
```

Comment:
This should be "The other option is to use"?  And (to avoid a separate email for each typo)
. line 25: "It should noted" -> "be noted"
. iine 43: "you can replicate with issues" should be "no issues"?